herddb-core/src/test/java/herddb/sql/CalcitePlannerTest.java

```diff
@@ -463,6 +460,76 @@ public void explainPlanTest() throws Exception {
         }
     }
 
+    @Test
+    public void showCreateTableTest() throws Exception {
```

Comment:
Please also create a test for a table created in a transaction, without committing the transaction

Reply:
Yes will do.